invenio_rdm_records/services/review/service.py

```diff
@@ -143,18 +144,44 @@ def delete(self, identity, id_, revision_id=None, uow=None):
         return True
 
     @unit_of_work()
-    def submit(self, identity, id_, data=None, revision_id=None, uow=None):
-        """Submit record for review."""
-        # Get record and check permission
+    def submit(
+        self, identity, id_, data=None, direct_publish=None, revision_id=None, uow=None
+    ):
+        """Submit record for review or direct publish to the community."""
         draft = self.draft_cls.pid.resolve(id_, registered_only=False)
-        self.require_permission(identity, "update_draft", record=draft)
-
         # Preconditions
         if draft.parent.review is None:
             raise ReviewNotFoundError()
 
         # since it is submit review action, assume the receiver is community
         resolved_community = draft.parent.review.receiver.resolve()
+        if direct_publish:
```

Comment:
Shouldn't we change this to:
```
if direct_publish or can_direct_publish:
```

Cases:
1. the user forces the direct_publish
2. the user does not force it, but (s)he can direct publish